packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -205,6 +221,12 @@ export const TooltipPlugin: React.FC<TooltipPluginProps> = ({
   );
 };
 
+function handleMouseEvent(e: MouseEvent) {
+  if (e.target) {
+    (e.target as HTMLDivElement).classList.toggle('plot-active');
+  }
+}
+
```

Comment:
it'd probably get rid of this and do something like this above (more explicit than toggle and doesnt need to get target from event, since we know it's bound to `.over` already):

```js
    const plotMouseLeave = (e: MouseEvent) => {
      setCoords(null);
      setIsActive(false);
      plotCtx.plot.over.classList.remove('plot-active');
    };
```